scripts/validate-configs.js

```diff
@@ -399,25 +401,54 @@ function getFlatRules(flatConfig) {
   }, {});
 
   // Sort the flat rules alphabetically and return them
-  return sortObject(rawFlatRules);
+  return normalizeRules
+    ? sortObject(rawFlatRules, normalizeRuleConfigValue)
+    : sortObject(rawFlatRules);
 }
 
 /**
  * Sorts the keys of the given object, inserts them in that order in a new
  * object, and returns that object.
  *
  * @param {Record<string, unknown>} obj - The object to sort.
+ * @param {Function} [valueNormalizer] - A function that takes a value and
+ * returns a "normalized" version of it. The value of every key on the sorted
+ * object will be passed through this function, if present.
  * @returns {Record<string, unknown>} The sorted object.
  */
-function sortObject(obj) {
+function sortObject(obj, valueNormalizer) {
```

Comment:
Nit: This no longer seems like an appropriate name for this function :thinking: 

Maybe it could be renamed to `reformatObject`. Or maybe the normalization can be moved to a separate function?

Reply:
Renamed to `normalizeObject` here: 282096f